en/Obsidian/Obsidian.md

```diff
@@ -51,7 +51,7 @@ On top of that, you can add isolated, independent building blocks to add feature
 
 We don't expect plugins to be the silver bullet to solve all the problems, but it does get us closer than a monolith solution, without all the features that you don't need cluttering the interface.
 
-We support [[Customizing CSS|custom CSS]] and have a [[Community plugins|beta plugin API]], which will enable even more interface customization and support less common use cases.
+We support [[CSS snippets|custom CSS]] and [[Community plugins]], which enable even more interface customization and support less common use cases.
```

Comment:
"custom CSS" is the legacy terminology, I don't mind just saying "CSS snippets and themes" here